src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,383 @@
+---
+title: First Discovery Server
+layout: default
+category: Overview
+---
+
+The First Discovery Server acts as a web server for a  [First Discovery Editor](https://github.com/GPII/first-discovery) instance and provides a means for storing preferences to the [GPII](http://gpii.net) [Preferences Server](https://github.com/GPII/universal). While the First Discovery Server is built with [gpii-express](https://github.com/gpii/gpii-express) and is not actually a [Kettle](https://github.com/amb26/kettle/blob/KETTLE-32/README.md) application, it does makes use of Kettle features for creating [datasources](https://github.com/amb26/kettle/blob/KETTLE-32/README.md#datasources), [configuration](https://github.com/amb26/kettle/blob/KETTLE-32/README.md#structure-of-a-kettle-config), and for [starting](https://github.com/amb26/kettle/blob/KETTLE-32/README.md#starting-a-kettle-application) the server.
```

Comment:
I don't think the form of explanation is quite helpful - describing a thing which is "built with Kettle but is not a Kettle app". I think it might be clearer to see that it "uses the Kettle configuration system but doesn't use Kettle's facilities for expressing HTTP servers"